docker/rabbitmq.conf

```diff
@@ -1 +1,5 @@
 management.load_definitions = /etc/rabbitmq/definitions.json
+
+# Allow the "guest" user to be used for convenience in the Docker development environment. Not advised in prod for
+# obvious reasons.
+loopback_users = none
```

Comment:
3.10 (and maybe 3.9? Haven't tried it.) doesn't require this configuration for whatever reason. Could be that it's Docker-aware, or could be that they just ship this configuration by default now in the currently-supported RabbitMQ images.